collections/_article/godot-game-awards-stayathomedev.md

```diff
@@ -0,0 +1,57 @@
+
+The winners will be announced in a premiere video on December 28th at 2:00pm EST on my channel so that everyone can watch together and help support the great work by these developers.
+
+
+### Tell us about your YouTube channel—what kind of content do you focus on, and what motivated you to start creating videos?
+I started [StayAtHomeDev](https://www.youtube.com/@stayathomedev) over two years ago to share my passion for creating through game development.  At the time, I had a potential job offer to be the Web Coordinator for a professional football team in the US.  But I had just been given a grant by Meta for some VR development work with their then Oculus Quest 2 headset so I used that to launch the channel. It really was an exercise to create independent income so I could stay at home with my kids.  That was a lifestyle choice I knew I wanted to make for them, so I took a shot at Youtube.
+
+I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/8nsJw89FG_A" frameborder="0" allowfullscreen style="width: 100%; aspect-ratio: 16 / 9; height: auto;"></iframe>
+
+### How did you discover Godot? When did you start using it? Do you have prior experience with other game engines?
+If you look at the earliest videos on the channel, you’ll notice they were all focused on the Unreal Engine.  I actually worked within Unreal first.  Long ago…like long ago…I started making little games with an engine called 3D GameStudio.
+
+When I started the channel, I knew of Godot (which I thought was called GoDot) but never tried it.  After spending more time on Youtube and lurking around other game dev content, Godot kept popping up, so I took it for a test drive.  I ended up making a video about it which went “viral” for a small channel while my first kid was being born.  There was clearly a lot of interest in the engine so I kept playing around with it and fell in love.  Not only with how streamlined it felt but also because it was open source and had such a strong sense of community and pride.  I decided I wanted to help support that with the channel.
```

Comment:
```suggestion
When I started the channel, I knew of Godot (which I thought was called GoDot) but never tried it.  After spending more time on YouTube and lurking around other game dev content, Godot kept popping up, so I took it for a test drive.  I ended up making a video about it which went “viral” for a small channel while my first kid was being born.  There was clearly a lot of interest in the engine so I kept playing around with it and fell in love.  Not only with how streamlined it felt but also because it was open source and had such a strong sense of community and pride.  I decided I wanted to help support that with the channel.
```